k8s/regions/oregon/prod-oregon-b.yaml

```diff
@@ -0,0 +1,102 @@
+kubernetes:
+    app_name: "sumo-prod"
+    namespace: "sumo-prod"
+    target_environment: "prod"
+    nodeport_name: "sumo-nodeport"
+    secrets_name: "sumo-secrets-prod"
+    image:
+        repo: "mozmeao/kitsune"
+        tag: "full-latest"
+        pull_policy: "Always"
+    # default values
+    limits:
+        memory_limit: 1Gi
+        memory_request: 256Mi
+        cpu_limit: 500m
+        cpu_request: 100m
+    replicas: 1
+    rollouts:
+        max_surge: 1
+        max_unavailable: 0
+    readiness: False
+    liveness: False
+    # app specific values
+    apps:
+        cron:
+            command: "./bin/run-cron.sh"
+            deployment_name: "sumo-prod-cron"
+            replicas: 0
+        celery:
+            command: "./bin/run-celery-worker.sh"
+            deployment_name: "sumo-prod-celery"
+            replicas: 0
+        web:
+            command: "./bin/run-prod.sh"
+            deployment_name: "sumo-prod-web"
+            replicas: 3
+            rollouts:
+                max_surge: 3
+                max_unavailable: 0
+            readiness:
+                failure_threshold: 2
+                initial_delay_seconds: 20
+                period_seconds: 5
+                success_threshold: 1
+                timeout_seconds: 5
+            liveness:
+                failure_threshold: 2
+                initial_delay_seconds: 20
+                period_seconds: 5
+                success_threshold: 1
+                timeout_seconds: 5
+
+app:
+    allowed_cidr_nets: SECRET
+    allowed_hosts: "prod.sumo.moz.works,support.mozilla.com,support.mozilla.org"
```

Comment:
For each env, we'll need to add one of:
`prod-oregon-a.sumo.moz.works`, `prod-oregon-b.sumo.moz.works`, `prod-frankfurt.sumo.moz.works`,
`stage-oregon-a.sumo.moz.works`, `stage-oregon-b.sumo.moz.works`, `stage-frankfurt.sumo.moz.works`